docs/guide/en/accordion.md

```diff
@@ -0,0 +1,210 @@
+# Accordion
+
+The Accordion widget renders a Bootstrap 5 accordion component within the **Yii framework**.
```

Comment:
How about adding screenshots to each guide to make widgets immediately recognizable?

Reply:
If we can add the images that Bootstrap 5 already has in its documentation, there's no problem, although my idea is clear and simple documentation that addresses all the topics explicitly.

Reply:
fixed.